js/src/layers/LayerGroup.js

```diff
@@ -10,6 +10,8 @@ export class LeafletLayerGroupModel extends layer.LeafletLayerModel {
     return {
       _view_name: 'LeafletLayerGroupView',
       _model_name: 'LeafletLayerGroupModel',
+      _view_module: 'jupyter-leaflet',
+      _model_module: 'jupyter-leaflet',
```

Comment:
I guess this may not actually be enough, do we also need the `_view_module_version` and the `_model_module_version`?

Reply:
Yes we might need them :)

Reply:
Do I just initialize them to null?